docs/labs/EclipseLoot.md

```diff
@@ -211,10 +220,8 @@ What's also handy though
 is adding an ```if``` statement to your code specifically to help you debug,
 something that says
 
-```java
-if (i == 67) {
-    System.out.println("reached 67th time");
-```
+    if (i == 67) {
```

Comment:
please reset this to be enclosed in a java code block again

Reply:
I reset it, but the majority of checks get denied with this change